sky/utils/common_utils.py

```diff
@@ -20,7 +20,7 @@
 
 
 def get_run_id():
-    """Returns a unique run id for this logging."""
+    """Returns a unique run id for each run of the program."""
```

Comment:
```
    """Returns a unique run id for each 'run'.

    A run is defined as the lifetime of a process that has imported `sky` and has called its CLI or programmatic APIs. For example, two successive `sky launch` are two runs."""
```
Should define "run" as it's non-obvious.  Is the "or programmatic APIs" part accurate?